Dockerfile-run

```diff
@@ -1,155 +1,165 @@
         make && \
-        make install && \
-        rm -rf ${DIR} && \
-#RUN  \
-## xvid https://www.xvid.com/
-        DIR=$(mktemp -d) && cd ${DIR} && \
+        make install
+### xvid https://www.xvid.com/
+RUN \
+        DIR=/tmp/xvid && \
+        mkdir -p ${DIR} && \
+        cd ${DIR} && \
         curl -sLO http://downloads.xvid.org/downloads/xvidcore-${XVID_VERSION}.tar.gz && \
         echo ${XVID_SHA256SUM} | sha256sum --check && \
         tar -zx -f xvidcore-${XVID_VERSION}.tar.gz && \
         cd xvidcore/build/generic && \
-        ./configure --prefix="${SRC}" --bindir="${SRC}/bin" --datadir="${DIR}" --disable-static --enable-shared && \
+        ./configure --prefix="${PREFIX}" --bindir="${PREFIX}/bin" --datadir="${DIR}" --enable-shared --enable-shared && \
         make && \
-        make install && \
-        rm -rf ${DIR} && \
-#RUN  \
-## fdk-aac https://github.com/mstorsjo/fdk-aac
-        DIR=$(mktemp -d) && cd ${DIR} && \
+        make install
+### fdk-aac https://github.com/mstorsjo/fdk-aac
+RUN \
+        DIR=/tmp/fdk-aac && \
+        mkdir -p ${DIR} && \
+        cd ${DIR} && \
         curl -sL https://github.com/mstorsjo/fdk-aac/archive/v${FDKAAC_VERSION}.tar.gz | \
         tar -zx --strip-components=1 && \
         autoreconf -fiv && \
-        ./configure --prefix="${SRC}" --disable-static --datadir="${DIR}" && \
+        ./configure --prefix="${PREFIX}" --enable-shared --datadir="${DIR}" && \
         make && \
-        make install && \
-        make distclean && \
-        rm -rf ${DIR} && \
-#RUN  \
-## ffmpeg https://ffmpeg.org/
-        DIR=$(mktemp -d) && cd ${DIR} && \
+        make install
+### ffmpeg https://ffmpeg.org/
+RUN \
+        DIR=/tmp/ffmpeg && \
+        mkdir -p ${DIR} && \
+        cd ${DIR} && \
         gpg --keyserver ha.pool.sks-keyservers.net --recv-keys ${FFMPEG_KEY} && \
         curl -sLO http://ffmpeg.org/releases/ffmpeg-${FFMPEG_VERSION}.tar.gz && \
         curl -sLO http://ffmpeg.org/releases/ffmpeg-${FFMPEG_VERSION}.tar.gz.asc && \
         gpg --batch  --verify ffmpeg-${FFMPEG_VERSION}.tar.gz.asc ffmpeg-${FFMPEG_VERSION}.tar.gz && \
         tar -zx --strip-components=1 -f ffmpeg-${FFMPEG_VERSION}.tar.gz && \
         ./configure \
-        --bindir="${SRC}/bin" \
         --disable-debug \
         --disable-doc \
         --disable-ffplay \
-        --disable-static \
+        --enable-shared \
         --enable-avresample \
-        --enable-gpl \
         --enable-libopencore-amrnb \
         --enable-libopencore-amrwb \
-        --enable-libfdk_aac \
         --enable-libmp3lame \
         --enable-libopus \
         --enable-libtheora \
         --enable-libvorbis \
         --enable-libvpx \
-        --enable-libx264 \
         --enable-libx265 \
         --enable-libxvid \
-        --enable-nonfree \
-        --enable-openssl \
+        --enable-gpl \
+        --enable-libx264 \
+        #--enable-nonfree \
+        #--enable-openssl \
```

Comment:
the removal of non-free because of https://trac.ffmpeg.org/wiki/CompilationGuide/Quick/libfdk-aac, caused the build to yell at me because of openssl (https://people.gnome.org/~markmc/openssl-and-the-gpl.html) 

Commented out so it can easily be enabled if needed